### PR TITLE
adapt to new kuksa repos

### PR DESCRIPTION
**Pull request by MP91**
## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: content/en/docs/concepts/development_model/vehicle_app_manifest/interfaces/vehicle_signal_interface.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The _Vehicle Signal Interface_ formerly known as [_Vehicle Model_](/docs/concept
 
 If a _Vehicle App_ requires a `vehicle-signal-interface`, it will act as a consumer of datapoints already available in the system. If, on the other hand, a _Vehicle App_ provides a `vehicle-signal-interface`, it will act as a provider (formerly feeder in KUKSA terms) of the declared datapoints.
 
-Furthermore, in the source code generated by this functional interface, a connection to [Kuksa Databroker](https://github.com/eclipse/kuksa.val/tree/master/kuksa_databroker) will be established via the configured Velocitas middleware. It uses the `broker.proto` if provided by the Kuksa Databroker to connect via gRPC. A seperate declaration of a `grpc-interface` for the databroker is **NOT** required. 
+Furthermore, in the source code generated by this functional interface, a connection to [Kuksa Databroker](https://github.com/eclipse-kuksa/kuksa-databroker) will be established via the configured Velocitas middleware. It uses the `broker.proto` if provided by the Kuksa Databroker to connect via gRPC. A seperate declaration of a `grpc-interface` for the databroker is **NOT** required.
```
Comment (dennismeister93) [inline content/en/docs/concepts/development_model/vehicle_app_manifest/interfaces/vehicle_signal_interface.md:20]:
```suggestion
Furthermore, in the source code generated by this functional interface, a connection to [KUKSA Databroker](https://github.com/eclipse-kuksa/kuksa-databroker) will be established via the configured Velocitas middleware. It uses the `broker.proto` if provided by the KUKSA Databroker to connect via gRPC. A seperate declaration of a `grpc-interface` for the databroker is **NOT** required.
```

File: content/en/docs/about/repository-overview/_index.md [hunk 2]
```diff
@@ -19,8 +19,8 @@ description: >
...
-| [kuksa.val.feeders](https://github.com/eclipse/kuksa.val.feeders/tree/main/dbc2val)               | The KUKSA DBC Feeder is a generic data feeder that reads data from the vehicle's [CAN bus](https://en.wikipedia.org/wiki/CAN_bus) defined by a DBC file, maps them to a set of data points (e.g. according to the VSS) and feeds it into the Data Broker.                                                                                                                                                                           |
+| [kuksa-databroker](https://github.com/eclipse-kuksa/kuksa-databroker)                    | Is a part of the _Vehicle Abstraction Layer (VAL)_ of the [Eclipse KUKSA project](https://github.com/eclipse-kuksa) and provides the KUKSA Data Broker (aka Vehicle Data Broker). The KUKSA Data Broker offers data points available in the vehicle to the _Vehicle Apps_ semantically aligned to a data model like the [Vehicle Signal Specification (VSS)](https://covesa.github.io/vehicle_signal_specification/).                                               |
```
Comment (dennismeister93) [inline content/en/docs/about/repository-overview/_index.md:22]: lets be consistent with "Databroker" on all occurences just as spelled in https://github.com/eclipse-kuksa/kuksa-databroker

**Review by dennismeister93 — CHANGES_REQUESTED** [review]

File: content/en/docs/concepts/development_model/vehicle_app_manifest/interfaces/vehicle_signal_interface.md (hunk 1 above)
Comment (MP91) [reply content/en/docs/concepts/development_model/vehicle_app_manifest/interfaces/vehicle_signal_interface.md]: done

File: content/en/docs/about/repository-overview/_index.md (hunk 2 above)
Comment (MP91) [reply content/en/docs/about/repository-overview/_index.md]: done

**Review by dennismeister93 — APPROVED** [review]
lgtm

**Pull request merged**